src/cli.ts

```diff
@@ -35,19 +46,26 @@ cli
 
 cli
   .command('', 'Start dev inspector')
-  .option('--config <configFile>', 'Config file path', { default: process.env.ESLINT_CONFIG })
+  .option('--config <configFile>', 'Config file path')
+  .option('--root <rootPath>', 'Root directory to inspect files. Default to directory of config file if not provided')
```

Comment:
@nzakas I didn't see ESLint has a similar flag. In #25 you mentioned `--basePath`, which to me could be a bit confusing as `base` often refers to URL base in web servers (https://vitejs.dev/guide/build#public-base-path).

So I picked `--root` - What's your opinion?

Reply:
When we talk about config files, all of the paths inside of `files` and `ignores` are resolved relative to a [base path](https://github.com/eslint/eslint/blob/main/lib/config/flat-config-array.js#L95). That's the term we decided on, so it's helpful to use it throughout.

When ESLint searches for and finds a config file on its own (`eslint.config.js`), then the base path is the directory in which the config file exists. So if the config file is `/path/to/eslint.config.js` then the base path is `/path/to`.

When someone passes [`-c` or `--config`](https://eslint.org/docs/latest/use/command-line-interface#-c---config) with a config file name, then the base path is the current working directory.